drivers/pcie/host/shell.c

```diff
@@ -79,7 +79,7 @@ static void show_bars(const struct shell *shell, pcie_bdf_t bdf)
 			      PCIE_CONF_BAR_IO(data) ? "I/O" : "MEM",
 			      PCIE_CONF_BAR_64(data) ? ", 64-bit" : "");
 
-		shell_fprintf(shell, SHELL_NORMAL, " addr ");
+		shell_fprintf(shell, SHELL_NORMAL, " addr 0x");
```

Comment:
this patch is correcting a previous patch in the same PR, so merge it directly with the original one.